docs/docs/end-users/GeneratingCode.fsx

```diff
@@ -32,83 +32,63 @@ In simple scenarios this can work out, but in the long run it doesn't scale well
 To illustrate the API, lets generate a simple value binding: `let a = 0`.
 *)
 
-#r "nuget: Fantomas.Core, 5.*" // Note that this will also load Fantomas.FCS, which contains the syntax tree types.
+#r "../../../src/Fantomas/bin/Release/net6.0/Fantomas.FCS.dll"
+#r "../../../src/Fantomas/bin/Release/net6.0/Fantomas.Core.dll" // In production use #r "nuget: Fantomas.Core, 6.0-alpha-*"
 
 open FSharp.Compiler.Text
-open FSharp.Compiler.Xml
-open FSharp.Compiler.Syntax
-open FSharp.Compiler.SyntaxTrivia
+open Fantomas.Core.SyntaxOak
 
 let implementationSyntaxTree =
-    ParsedInput.ImplFile(
-        ParsedImplFileInput(
-            "filename.fsx",
-            true,
-            QualifiedNameOfFile(Ident("", Range.Zero)),
-            [],
-            [],
-            [ SynModuleOrNamespace(
-                  [],
-                  false,
-                  SynModuleOrNamespaceKind.AnonModule,
-                  [ SynModuleDecl.Let(
-                        false,
-                        [ SynBinding(
-                              None,
-                              SynBindingKind.Normal,
-                              false,
-                              false,
-                              [],
-                              PreXmlDoc.Empty,
-                              SynValData(None, SynValInfo([], SynArgInfo([], false, None)), None),
-                              SynPat.Named(SynIdent(Ident("a", Range.Zero), None), false, None, Range.Zero),
-                              None,
-                              SynExpr.Const(SynConst.Int32(0), Range.Zero),
-                              Range.Zero,
-                              DebugPointAtBinding.Yes Range.Zero,
-                              { EqualsRange = Some Range.Zero
-                                InlineKeyword = None
-                                LeadingKeyword = SynLeadingKeyword.Let Range.Zero }
-                          ) ],
-                        Range.Zero
-                    ) ],
-                  PreXmlDoc.Empty,
-                  [],
-                  None,
-                  Range.Zero,
-                  { LeadingKeyword = SynModuleOrNamespaceLeadingKeyword.None }
-              ) ],
-            (false, false),
-            { ConditionalDirectives = []
-              CodeComments = [] },
-            Set.empty
-        )
+    Oak(
+        [],
+        [ ModuleOrNamespaceNode(
+              None,
+              [ BindingNode(
+                    None,
+                    None,
+                    MultipleTextsNode([ SingleTextNode("let", Range.Zero) ], Range.Zero),
+                    false,
+                    None,
+                    None,
```

Comment:
I know this was already merged and it's totally fine, but I think it would be helpful to use explicit param names here, the waterfall of `None` could feel really confusing for some (myself included lol) and also serve to document what the params actually represent

Reply:
I wouldn't mind accepting a PR for that if you feel strongly about it.
For better or worse you can see the signature when you hover over it:
![image](https://user-images.githubusercontent.com/2621499/219301901-e199b9af-c596-40f0-b313-e6cf98d7ce29.png)

Reply:
Ooh I did not know that, that's super useful! I think it would still be helpful to have param names just to illustrate what the parameters are in examples but that's good to know!

Reply:
As I said 😉

> I wouldn't mind accepting a PR